packages/wonder-blocks-form/src/components/radio-group.tsx

```diff
@@ -149,13 +154,15 @@ export default class RadioGroup extends React.Component<RadioGroupProps> {
                             groupName: groupName,
                             id: `${groupName}-${value}`,
                             key: value,
-                            onChange: () => this.handleChange(value),
+                            onChange: () => handleChange(value),
```

Comment:
**nit:** Out of scope for this PR, but I noticed that `handleChange` could go away and instead we could assign directly the prop:

```suggestion
                            onChange: onChange,
```